amethyst_renderer/src/system.rs

```diff
@@ -89,14 +89,7 @@ where
         );
     }
 
-    fn do_render(
-        &mut self,
-        (mut event_handler, mut window_messages, mut screen_dimensions, data): RenderData<P>,
-    ) {
-        #[cfg(feature = "profiler")]
-        profile_scope!("render_system");
-        use std::time::Duration;
-
+    fn do_window_management(&mut self, (mut window_messages, mut screen_dimensions): WindowData) {
```

Comment:
Nice.

Reply:
I needed it in order to access the ScreenDimensions in a pass :P

Reply:
Yeah, I realised that. But it also improved the state of `RenderSystem`, so it's all good :)